src/ViewInstance.js

```diff
@@ -156,6 +157,39 @@ class ViewInstance extends React.Component {
     return ref || {};
   }
 
+  getActionMenu({ onToggle }, instance) {
```

Comment:
It's a little weird that we have to pass that instance around and that it's defined in `render()`, but that's a problem for another day.

Reply:
I commented on this below. I would like your input 👍